docs/docs/20-quickstart.mdx

```diff
@@ -613,9 +613,8 @@ thereby forming a _pipeline_:
 1. `uat` subscribes to `test`
 1. `prod` subscribes to `uat`.
 
-We leave it as an exercise to the reader to use the `kargo stage promote`
-command to progress the `Freight` from `stage` to `uat` and again from `uat` to
-`prod`.
+We leave it as an exercise to the reader to use the `kargo promote` command
+to progress the `Freight` from `stage` to `uat` and again from `uat` to `prod`.
```

Comment:
```suggestion
to progress the `Freight` from `test` to `uat` and again from `uat` to `prod`.
```

This wasn't your mistake, but it's a good time to fix it.

Reply:
Ah, great 🎣. Fixed!